dockers/docker-orchagent-cavm/copp.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
Remove this file. sonic will use the same https://github.com/Azure/sonic-swss/blob/master/swssconfig/sample/copp.config.json across different platforms.